statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -4031,8 +4109,12 @@ def summary(self, alpha=.05, start=None, title=None, model_name=None,
             ('No. Observations:', [self.nobs]),
             ('Log Likelihood', ["%#5.3f" % self.llf]),
         ]
-        if hasattr(self, 'rsquared'):
-            top_right.append(('R-squared:', ["%#8.3f" % self.rsquared]))
+        if hasattr(self, 'rsquared_rwdrift'):
+            top_right.append(('R-squared:',
+                              ["%#8.3f" % self.get_rsquared('rwdrift')[0]]))
+        if hasattr(self, 'rsquared_mean'):
```

Comment:
I think by default we should only show the `rwdrift` version

Reply:
done